docs/tutorials/excel-tutorial-create-custom-functions.md

```diff
@@ -0,0 +1,203 @@
+ms.date: 09/05/2018
+ms.topic: tutorial
+#Customer intent: As an add-in developer, I want to create a custom function in Excel to increase productivity. 
+---
+# Create a streaming Excel custom function
+
+## Introduction
+
+Custom Functions give you the power to automate processes in Excel. You can use them for something as simple as creating your own custom calculation, similar to `=SUM()` or for more complex tasks, like streaming data from the web in real-time right into your worksheet.
+
+In this tutorial, you will learn how to:
+> [!div class="checklist"]
+> * Create a custom function project using yo office
+> * Build a custom function which performs a simple calculation
+> * Use custom functions to request data from the web
+> * Create a custom function which streams real-time data from the web
+
+## Prerequisites
+
+* [Node.js and npm](https://nodejs.org/en/)
+* [Git Bash](https://git-scm.com/downloads) (or another Git client)
+* [Yeoman](http://yeoman.io/) and the [Yo Office generator](https://www.npmjs.com/package/generator-office)
+* Office 2016 for Windows and [join the Office Insider program](https://products.office.com/office-insider). You must have Office build number 8711 or later.
+
+## Create your add-in project
+
+You’ll begin this tutorial by using the Yo Office Yeoman generator, which will automatically populate the files you need for your project.
+
+    1. In your command line interface, create a scaffold of your project (by default, this should be in your C:\Users\YourUserName folder).
+
+    ```bash
+    yo office
+    ```
+
+    ![Yo Office bash prompts for custom functions](../images/yo-office-excel-cfs-stock-ticker.png)
+    Answer the prompts as directed below:
+
+    * Choose a project type: `Excel Custom Functions Add-in project (September 2018 Preview Refresh: Requires the Insider channel for Excel)`
+    * What do you want to name your add-in? `stock-ticker`
+
+    For this tutorial, choose Javascript as the language you would like to use to build your add-in.
+
+    After you complete the wizard, the generator will create the project files and install supporting Node components.
+
+    2. Next, start a localhost instance by running one of the below commands in your command line interface.
+
+    If you are developing using the desktop version of Excel, use:
+
+    ```bash
+    npm start
+    ```
+
+    If you are using Excel Online, use:
+
+    ```bash
+    npm start-web
+    ```
+
+    3. You will also need to register your custom-functions add-in. In Excel, select **Insert > My Add-ins > Insert an Add-in**. This will bring up a list of available add-ins. Under "Developer Add-ins" you will see your add-in, under the name "Excel Custom Function". Select it to register it.
+
+    Select **Insert > Add-ins**. Choose **Manage My Add-ins** and select **Upload My Add-in**. Click "Browse..." for your manifest file (`C:\Users\LabUser\Stock Ticker\manifest.xml`), then click Open, select **Upload**. 
```

Comment:
These navigation steps, which appear in 3 places in this article, are no longer correct. The steps are now **File** > **Extensions** > **My Add-ins** tab. Then **Manage My Add-ins**.

Reply:
Hrm ok! Do you know if you are part of any experiments/flights? I had to go through and disable all experiments at some point, I think.

Reply:
I'm on the dogfood tool, DevMain channel, which updates Office 4 or 5 times a week. I don't know if I'm on any experiments. If you're uncertain, then leave things the way they work on your version. If necessary, we can change the instructions later.

Reply:
Ok, let's go with that for now? @keyur32 just letting you know there may or may not be a problem with the instructions for this tutorial.